stl/src/vector_algorithms.cpp

```diff
@@ -2249,6 +2250,110 @@ namespace {
 #endif // ^^^ !defined(_M_ARM64EC) ^^^
         return __std_minmax_impl<_Mode, typename _Traits::_Scalar, _Sign>(_First, _Last);
     }
+
+
+    template <class _Traits, class _Ty>
+    const void* __std_is_sorted_until_impl(const void* _First, const void* const _Last, const bool _Greater) noexcept {
+        const ptrdiff_t _Left_off  = 0 - static_cast<ptrdiff_t>(_Greater);
+        const ptrdiff_t _Right_off = static_cast<ptrdiff_t>(_Greater) - 1;
+
+        if constexpr (_Traits::_Vectorized) {
+#ifndef _M_ARM64EC

```

Comment:
This `if constexpr` is empty for ARM64EC - it seems that the guards should be moved to encompass it entirely.

Reply:
It is better to `static_assert(false)` here, like in two other occurrences.